### PR TITLE
Handle right click on a bell

### PR DESCRIPTION
**Pull request by Arkobalt**
My first PR ever, don't really know how to structure it.
The feature is quite simple so i dont really need to explain it, i guess.

### CODE REVIEW

File: src/player/bells.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      if (block.name === 'bell') {
+        switch (
+          direction // bell face clicked
+        ) {
+          case 2: // north
+            client.write('block_action', {
+              location,
+              byte1: 1,
+              byte2: 2,
+              blockId: 677,
```
Comment (Sceat) [inline src/player/bells.js:19]: those 3 values are magic numbers

File: src/context.js [hunk 2]
```diff
@@ -267,6 +268,8 @@ export async function observe_client(context) {
   player_respawn.observe(context)
   player_heartbeat.observe(context)
 
+  player_bells.observe(context) // fezrhnjhifhbredjihebghfrvdihje
```
Comment (Sceat) [inline src/context.js:271]: `fezrhnjhifhbredjihebghfrvdihje` ?

**Review by Sceat — CHANGES_REQUESTED** [review]
Thank for contributing to AresRPG !

About your PR, you have a huge code duplication, you could simply
```js
client.write('block_action', {
    location,
    byte1: 1,
    byte2: direction,
    blockId: 677,
  })
  play_sound({
    client,
    sound: 'block.bell.use',
    category: CATEGORY.AMBIENT,
    ...location,
  })
```

I guess creating a file just for the bells are fine @DeltaEvo ? but maybe it should be in a more generic world entity files ? like trapdoors etc (cc @KalvinVilla)

File: src/player/bells.js (hunk 1 above)
Comment (Arkobalt) [reply src/player/bells.js:19]: I didn't think about it x).
The slight difference with this version is that the bell ring (sound) even when there is no animation (when rang from top or bottom).
I son't know the exact details of what was required but maybe we'll need to add another condition for sound if done this way.

File: src/context.js (hunk 2 above)
Comment (Arkobalt) [reply src/context.js]: It took me a day to realise that this was about the comment.
It's fixed.

File: src/player/bells.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          byte2: direction, // handle side from wich the bell is clicked (0 et 1 for up and down and other number for each side)
+          blockId: 677, // bell id
+        })
+        play_sound({
+          client,
+          sound: 'block.bell.use',
+          category: CATEGORY.AMBIENT,
+          x: location.x,
+          y: location.y,
+          z: location.z,
```
Comment (Sceat) [inline src/player/bells.js:23]:
```suggestion
          ...location
```

**Review by Sceat — CHANGES_REQUESTED** [review]

File: src/player/bells.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  observe(context) {
+    const { client, world } = context
+    client.on('block_place', async ({ location, direction }) => {
+      const block = await get_block(world, location)
+      if (block.name === 'bell' && direction >= 2) {
+        client.write('block_action', {
+          location,
+          byte1: 1, // only number 1 is available for bell animation
+          byte2: direction, // handle side from wich the bell is clicked (0 et 1 for up and down and other number for each side)
+          blockId: 677, // bell id
```
[the remark this answers is not in the document]
Comment (Arkobalt) [reply src/player/bells.js:15]: oh, i see, i'll do that now

**Comment by Sceat** [comment]
Seems good, let's now wait for @DeltaEvo opinion on https://github.com/aresrpg/aresrpg/pull/572#pullrequestreview-1205052568

**Comment by Arkobalt** [comment]
My last commit was helpfull for test but may cause probleme because the mobs.json file from floor1 when starting the server. I dont know how without requesting a pull on floor1 but modifications on mobs.json should be discarded or the last commit should be ignored.

**Review by Sceat — APPROVED** [review]

**Pull request merged**